pkg/kritis/review/validating_transport.go

```diff
@@ -69,16 +69,24 @@ func (avt *AttestorValidatingTransport) GetValidatedAttestations(image string) (
 		return nil, err
 	}
 	for _, a := range attestations {
-		decoded_sig, err := base64.StdEncoding.DecodeString(a.Signature)
-		if err != nil {
-			glog.Infof("Cannot base64 decode signature: %v", err)
-			continue
+		if a.SignatureType == metadata.GenericSignatureType || a.SignatureType == metadata.UnknownSignatureType {
+			return nil, fmt.Errorf("Signature type %s is not supported for Attestation %v", a.SignatureType.String(), a)
```

Comment:
Tests for this? I can remove the tests once other types are supported.

Reply:
Done.